src/landing/Welcome.tsx

```diff
@@ -14,9 +21,16 @@ export class Welcome extends Component<{}> {
         {({ applications, loading }): ReactNode => {
           if (loading) {
             return (
-              <Bullseye>
+              <EmptyState
```

Comment:
I believe it would be better to put this into its own component.
We could, for example, create a `Loading` component and remove the `if (loading)` stuff, replacing it with:
`<Loading visible={ loading } />`

The loading component would return `null` if `visible` is false.
The new component should be placed in the `common` folder.